components/upload/index.tsx

```diff
@@ -2,4 +2,9 @@ import Upload from './Upload';
 import Dragger from './Dragger';
 
 Upload.Dragger = Dragger;
+
+function Test() {
+  return <Upload.Dragger />;
```

Comment:
This doesn't work as I expected, `Upload.Dragger` is any, so that we cannot get any tips in VSCode.